src/SprykerEco/Zed/Computop/Persistence/ComputopQueryContainerInterface.php

```diff
@@ -19,7 +19,7 @@ interface ComputopQueryContainerInterface
      *
      * @return \Orm\Zed\Computop\Persistence\SpyPaymentComputopQuery
      */
-    public function queryPaymentById($idPayment);
+    public function queryPaymentById(int $idPayment);
```

Comment:
please revert changes for this interface since they are bc-breaking and lead to major

Reply:
Fixed